src/inference/src/ie_core.cpp

```diff
@@ -10,6 +10,7 @@
 #include <memory>
 #include <mutex>
 #include <string>
+#include <threading/ie_executor_manager.hpp>
```

Comment:
need to use ""

Reply:
I don't think so because it is not a local file.